scripts/globals/mobskills/vulture_1.lua

```diff
@@ -0,0 +1,17 @@
+---------------------------------------------
+-- (Vulture 1)
```

Comment:
I know you inherited this from what was already in the db, but could we change this to be something a little more descriptive? `provoker_2hr` or something would probably work. That change would need to be reflected here, in this file's name and the entry in mob_skills

Reply:
@zach2good it's used by other mobs as well and that is what reports out from actionview in captures. changing the name will make finding and using it potentially difficult.

Reply:
Aha, makes sense! Disregard this comment!